packages/now-cli/test/integration.js

```diff
@@ -509,13 +539,20 @@ test('Deploy `api-env` fixture and test `vercel env` command', async t => {
   async function nowEnvAddSystemEnv() {
     const now = execa(
       binaryPath,
-      ['env', 'add', 'VERCEL_URL', ...defaultArgs],
+      ['env', 'add', 'system', 'VERCEL_URL', ...defaultArgs],
       {
         reject: false,
         cwd: target,
       }
     );
 
+    await waitForPrompt(
+      now,
+      chunk =>
+        chunk.includes('What’s the value of') && chunk.includes('VERCEL_URL')
+    );
+    now.stdin.write(`\n`); // select VERCEL_URL
```

Comment:
Why do we make the user select an empty value for system env vars?

Reply:
It shows a list of system env vars and we select the first one: `VERCEL_URL`